katcp/server.py

```diff
@@ -2237,13 +2250,25 @@ def request_sensor_sampling(self, req, msg):
             ?sensor-sampling cpu.power.on
             !sensor-sampling ok cpu.power.on none
 
-            ?sensor-sampling cpu.power.on period 500
-            !sensor-sampling ok cpu.power.on period 500
+            ?sensor-sampling cpu.power.on period 0.5
+            !sensor-sampling ok cpu.power.on period 0.5
+
+            if BULK_SET_SENSOR_SAMPLING is enabled then:
+
+            ?sensor-sampling cpu.power.on,fan.speed
+            !sensor-sampling fail Cannot\_query\_multiple\_sensors
+
+            ?sensor-sampling cpu.power.on,fan.speed period 0.5
+            !sensor-sampling ok cpu.power.on,fan.speed period 0.5
```

Comment:
Perhaps, update the docstring to match the actual implementation

Reply:
This does match the request and reply, but maybe you mean the informs should be included?  We could add that, and then the same for lines 2253-2254.